### ISSUE TITLE
[MNG-2208] Failure to checkout M2 module from SVN

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#4262]
**[Felipe Leme](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=felipeal)** opened **[MNG-2208](https://issues.apache.org/jira/browse/MNG-2208?redirect=false)** and commented

I have a M2 project with many sub-projects; one of them is the web application, whose directory is called webapp, the module on the parent's pom is also webapp but the project's artifactId is mycompany-web. When I run 'mvn install' from the parent on command line, it works fine; when I import the parent's pom into continuum, it works fine also (i.e., it automatically set the sub-projects). But continuum fails to build the webapp project with the message:

Provider message: The svn command failed.
Command output:

---

svn: URL 'http://svn.versiondude.net/mycompany/voxblue/trunk/mycompany-web' doesn't exist

In other words, continuum created the wrong URL: it used the sub-project's artifactId and not the name defined on the parent's POM (I'm not sure if that's a bug on continuum or M2, but the fact is that either both should work or both fail).

---

**Affects:** 2.0.3

**Comment by jira-importer** [comment]
**[Emmanuel Venisse](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=evenisse)** commented

ok, so it's a maven problem, i move this issue to MNG project
